### ISSUE TITLE
EfficientNet as a feature extractor for TF object detection

### ISSUE THREAD

**Issue by ldalzovo** [tensorflow/tpu#380]
Hello, is it possible to use EfficientNet as a feature extractor for TF object detection API?
Is there a guide, or can you provide it, with the required steps to bring this or other official tpu models in TF-OD?

Thank you!

**Comment by saberkun** [comment]
We have a pending change to use EfficientNet as the backbone for the TPU maskrcnn model in this repo.
For TF-OD, it requires some work. As we expose the endpoints required for detection framework, probably you can get a feature extractor too.

**Comment by jamessmith90** [comment]
@saberkun 
I understand that this is the TPU repository of Tensorflow but more people are in interested in having a GPU version of EfficientNet Object Detection. Clients don't want to put their data on cloud.

**Comment by dexception** [comment]
@saberkun 
No new work is being done on the GPU side of Tensorflow. 
Can you confirm ?

If so i would like to move to PyTorch. 
Not interested in TPU's !

**Comment by nanfangyuan** [comment]
> Hello, is it possible to use EfficientNet as a feature extractor for TF object detection API?
> Is there a guide, or can you provide it, with the required steps to bring this or other official tpu models in TF-OD?
> 
> Thank you!

but not downed the model??

**Comment by jamessmith90** [comment]
@saberkun 
Any progress on Efficient Mask-RCNN ?
